bin/storj.js

```diff
@@ -396,6 +396,9 @@ var ACTIONS = {
         return log('error', err.message);
       }
 
+      var keyring = storj.KeyRing(DATADIR, program.keypass);
```

Comment:
Use this function to get the keyring: https://github.com/aleitner/core/blob/13627304494900a1234b3644da30fcc68d213fec/bin/storj.js#L101